manifests/npm.pp

```diff
@@ -10,13 +10,15 @@
   $uninstall_options = [],
   $home_dir          = '/root',
   $user              = undef,
+  $use_package_json  = false,
 ) {
 
   validate_re($ensure, '^[^<>=]', "The ${module_name}::npm defined type does not accept version ranges")
   validate_array($install_options)
   validate_string($package)
   validate_absolute_path($target)
   validate_array($uninstall_options)
+  validate_bool($use_package_json)
```

Comment:
If you rebase, this should now use data types instead (remove `validate_bool`, and update
`$use_package_json  = false,`
to 
`Boolean $use_package_json  = false,`
e.g., https://github.com/voxpupuli/puppet-nodejs/blob/master/manifests/init.pp#L4

Reply:
True, but probably better to change all of these in a new PR, now this one